prac_09/silver_service_taxi.py

```diff
@@ -0,0 +1,19 @@
+"""
+SilverServiceTaxi class (extends Taxi)
+"""
+from prac_09.taxi import Taxi
+
+
+class SilverServiceTaxi(Taxi):
+    flagfall = 4.50
+
+    def __init__(self, name, fuel, fanciness):
+        """Initialise a SilverServiceTaxi instance."""
+        super().__init__(name, fuel)
+        self.price_per_km *= fanciness
+
+    def get_fare(self):
+        return round(super().get_fare() + self.flagfall, 1)
```

Comment:
The exercise is on inheritance, better to have round() in base Taxi class instead of here